lib/rouge/lexers/javascript.rb

```diff
@@ -206,7 +206,7 @@ class JSON < RegexLexer
       desc "JavaScript Object Notation (json.org)"
       tag 'json'
       filenames '*.json'
-      mimetypes 'application/json'
+      mimetypes 'application/json', 'application/vnd.api+json'
```

Comment:
@kylef good idea - I will do some exploring to see what it would look like to support a * for the different extensions.